x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -7,195 +7,176 @@
 import React, { useCallback, useMemo } from 'react';
 import styled from 'styled-components';
 import { i18n } from '@kbn/i18n';
-import {
-  htmlIdGenerator,
-  EuiButton,
-  EuiI18nNumber,
-  EuiKeyboardAccessible,
-  EuiFlexGroup,
-  EuiFlexItem,
-} from '@elastic/eui';
+import { htmlIdGenerator, EuiButton, EuiI18nNumber, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+// eslint-disable-next-line import/no-nodejs-modules
+import querystring from 'querystring';
 import { NodeSubMenu, subMenuAssets } from './submenu';
 import { applyMatrix3 } from '../lib/vector2';
-import { Vector2, Matrix3, AdjacentProcessMap, ResolverProcessType } from '../types';
-import { SymbolIds, useResolverTheme, NodeStyleMap, calculateResolverFontSize } from './assets';
+import { Vector2, Matrix3, AdjacentProcessMap } from '../types';
+import { SymbolIds, useResolverTheme, calculateResolverFontSize, nodeType } from './assets';
 import { ResolverEvent, ResolverNodeStats } from '../../../common/endpoint/types';
 import { useResolverDispatch } from './use_resolver_dispatch';
 import * as eventModel from '../../../common/endpoint/models/event';
-import * as processModel from '../models/process_event';
 import * as selectors from '../store/selectors';
+import { CrumbInfo } from './panels/panel_content_utilities';
+
+/**
+ * A map of all known event types (in ugly schema format) to beautifully i18n'd display names
+ */
+export const displayNameRecord = {
+  application: i18n.translate(
+    'xpack.securitySolution.enpoint.resolver.applicationEventTypeDisplayName',
```

Comment:
did you change these to `enpoint` on purpose, or did you and clippy get into a fight? @bkimmel can you review this before merge.

Reply:
derp: https://github.com/elastic/kibana/pull/69098/commits/ed73b46ab455e9106d34ce5cb2c947014a7d5a55